build-cmake/CMakeLists.txt

```diff
@@ -8,6 +8,8 @@ CMAKE_MINIMUM_REQUIRED(VERSION 3.10 FATAL_ERROR)
 cmake_policy(SET CMP0074 NEW)
 cmake_policy(SET CMP0091 NEW)
 cmake_policy(SET CMP0112 NEW)
+# TODO: Convert DART to CTEST
+cmake_policy(SET CMP0145 OLD)
```

Comment:
Can you drop or revert this commit from your branch?

This warning seems harmless on the newer Cmake in the GitHub ubuntu-22.04 image, but on the older Cmake that ships with Ubuntu 22.04 I was running into failures because of this `cmake_policy` line for some reason.

Reply:
I have added if(POLICY CMP0145) what do you think ?